File: api/src/main/java/br/com/vr/apis/requests/UnlockPurchaseCardRequest.java
package br.com.vr.apis.requests;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class UnlockPurchaseCardRequest {

    @JsonProperty("purchaseCardId") private String purchaseCardId;

    @JsonProperty("unlockValue") private Boolean unlockValue;

}

File: api/src/main/kotlin/br/com/vr/apis/requests/UnlockPurchaseCardRequest.kt
package br.com.vr.apis.requests

import com.fasterxml.jackson.annotation.JsonProperty

data class UnlockPurchaseCardRequest(
    @JsonProperty("purchaseCardId")
    val purchaseCardId: String?,
    @JsonProperty("unlockValue")
    val unlockValue: Boolean?
)